.doctrine-project.json

```
@@ -9,17 +9,27 @@
       "slug": "latest",
       "upcoming": true
     },
+    {
+      "name": "3.8",
+      "branchName": "3.8.x",
+      "slug": "3.8",
```

Should be `lastest` according to the convention in other projects.

```suggestion
      "slug": "latest",
```